internal/command/logs/shipper_setup.go

```diff
@@ -80,12 +78,21 @@ func runSetup(ctx context.Context) (err error) {
 		fmt.Fprintf(io.ErrOut, "Provisioning a log shipper VM in the app named %s\n", shipperApp.Name)
 	}
 
-	// Fetch or create the org-specific Logtail integration
+	// Fetch or create the Logtail integration for the app
 
+	var addOnName = appName + "-log-shipper"
 	getAddOnResponse, err := gql.GetAddOn(ctx, client, addOnName)
 
 	if err != nil {
```

Comment:
Would it be worth checking for a more specific error here?

Reply:
Will address in a follow-up as the gqclient library is a bit weird about errors.